OpenRA.Mods.Common/Widgets/RadarWidget.cs

```diff
@@ -63,8 +63,8 @@ public sealed class RadarWidget : Widget, IDisposable
 		PlayerRadarTerrain playerRadarTerrain;
 		Player currentPlayer;
 
-		public string SoundUp { get; private set; }
-		public string SoundDown { get; private set; }
+		public string SoundUp;
```

Comment:
These should probably move up with the other widget fields?

Do we have a convention for widget fields to be readonly like we do for traits?

Reply:
Moved upwards.

A quick flick through some widgets and it seems read-write fields are more common, but there are some read-only ones littered about.